namer/videophash.py

```diff
@@ -51,13 +60,25 @@ def __generate_image_thumbnail(self, file: Path) -> Optional[Image.Image]:
     def get_stash_phash(self, file: Path) -> Optional[imagehash.ImageHash]:
         return self.__execute_stash_phash(file)
 
+    def __download_stash_phash(self):
+        os = platform.system().lower()
+        url = f'https://github.com/DirtyRacer1337/stash_phash/releases/download/nightly/stash_phash-{os}.zip'
```

Comment:
it would be better to download from this in to the docker image and in GitHub actions directly.    you might move the project, link might die, it could become an attack vector.    you could also leave an issue for me in the future to handle this.

Reply:
Yeah, I know, it's more for testing, at-least you can start to play around with phash